main.go

```diff
@@ -137,3 +160,23 @@ func checkOSSIndex(purls []string, packageCount int) {
 		os.Exit(count)
 	}
 }
+
+func auditWithIQServer(purls []string, applicationID string, config configuration.Configuration) {
```

Comment:
The audit with IQ Server does not take into account any CVE exclusions that have been passed via the command line or define in .nancy-ignore file via the `config.CVEList`. Unless there is something I'm missing with Nexus IQ where that might not be wanted we should still support that feature. 

It might make sense to pull that up further. Right now we are doing it at the auditing level. Maybe we should pull that up into right after/during parsing of the purls so that all auditing code can take part in that without having to do it individually every time we add a new auditing mechanism.

Reply:
Right, so with Nexus IQ Server you wouldn't manage any of the CVE exclusions, etc... inside of `nancy` or your repo, that's all managed in Nexus IQ Server, including policy based on what you want to cause failures, waivers on what you think is something you can live with, etc...